apps/demos/Demos/DropDownBox/MultipleSelection/React/App.tsx

```diff
@@ -48,7 +86,7 @@ function App() {
         onItemSelectionChanged={treeViewItemSelectionChanged}
       />
     ),
-    [treeDataSource],
+    [syncTreeViewSelection, treeViewItemSelectionChanged],
```

Comment:
Is this intended?

Reply:
There was a warning from 'react-hooks/exhaustive-deps' rule.